### PR TITLE
Common regex patterns for log redaction

### PR DESCRIPTION
**Pull request by jorgebay**
Include regex for common PII patterns.

### CODE REVIEW

File: docs/REGEX.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+```yaml
+    - env:
+        - name: LOGDNA_REDACT_REGEX
+          value: (?:(\+(?:\d{1}|\d{2})\s)?\(?\d{3}\)?[\s.-]?\d{3}[\s.-]?\d{4})
+```
+
+## Exclude noisy messages or sensitive information
+
+Exclude noisy messages or sensitive information from logs.
```
Comment (alfeng6) [inline docs/REGEX.md:69]: I'd want us to make it super clear that this is for an entire log line.

File: docs/REGEX.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
+# Common regex patterns for log redaction and exclusion
+
+You can use `LOGDNA_REDACT_REGEX` environment variable to redact parts of the log line, replacing matches with
+[REDACTED]. Additionally, you can use `LOGDNA_LINE_EXCLUSION_REGEX` variable to define lines to exclude.
```
Comment (alfeng6) [inline docs/REGEX.md:4]:
```suggestion
[REDACTED]. Additionally, you can use `LOGDNA_LINE_EXCLUSION_REGEX` variable to define entire lines to exclude.
```

File: docs/REGEX.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+```yaml
+    - env:
+        - name: LOGDNA_REDACT_REGEX
+          value: (?:4[0-9]{12}(?:[0-9]{3})?|[25][1-7][0-9]{14}|6(?:011|5[0-9][0-9])[0-9]{12}|3[47][0-9]{13}|3(?:0[0-5]|[68][0-9])[0-9]{11}|(?:2131|1800|35\d{3})\d{11})
+```
+
+## Social Security numbers
+
+Redact social security numbers with dashes, spaces or all the digits together.
```
Comment (alfeng6) [inline docs/REGEX.md:39]:
```suggestion
Redact Social Security Numbers with dashes, spaces or all the digits together.
```

**Review by alfeng6 — CHANGES_REQUESTED** [review]
just a few changes — overall really great!

File: docs/REGEX.md (hunk 1 above)
Comment (jorgebay) [reply docs/REGEX.md:69]: Good point, we should make clear that the line will be excluded, how about:

```suggestion
Exclude the entire log line for noisy messages or sensitive information that doesn't belong in the logs.
```

?

Comment (alfeng6) [reply docs/REGEX.md:69]:
```suggestion
Exclude entire log lines with noisy messages or sensitive information.
```

**Review by alfeng6 — APPROVED** [review]
looks great!

**Pull request merged**